openmm/build.sh

```diff
@@ -1,5 +1,46 @@
 #!/bin/bash
 
+# Fix hbb issues
+if [ ! -e /opt/rh/devtoolset-2/root/usr/lib/gcc/x86_64-CentOS-linux/ ]; then
```

Comment:
this isn't the condition you want to check. Do you need a check like this at all? If yes, remove the if statement and change the next line to `ln -sf` but I can't see why that would be necessary if we start from a fresh image every time